apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -129,21 +159,78 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
         });
     };
 
+    const handleTOTPVerificationCodeSubmit = (event: React.FormEvent<HTMLFormElement> ) =>{
+
+        let verificationCode =  event.target[0].value;
+        for (let pinCodeIndex = 1;pinCodeIndex<6;pinCodeIndex++){
+            verificationCode = verificationCode + event.target[pinCodeIndex].value;
+        }
+        verifyCode(verificationCode);
+    };
+
+
+    /**
+     * Focus to next pin code after enter a value.
+     *
+     * @param {string} field The name of the field.
+     */
+    const focusInToNextPinCode = (field: string): void => {
+        switch (field) {
+            case "pincode-1":
+                pinCode2.current.focus();
+                break;
+            case "pincode-2":
+                pinCode3.current.focus();
+                break;
+            case "pincode-3":
+                pinCode4.current.focus();
+                break;
+            case "pincode-4":
+                    pinCode5.current.focus();
+                    break;
+            case "pincode-5":
+                pinCode6.current.focus();
+                break;
+        }
+    };
+
+    /**
+     * Focus to previous pin code after enter Backspace or Delete.
+     *
+     * @param {string} field The name of the field.
+     */
+    const focusInToPreviousPinCode = (field: string): void => {
```

Comment:
Fix the formatting issue in switch.
```suggestion
    const focusInToPreviousPinCode = (field: string): void => {

```